pkg/cmd/convert.go

```diff
@@ -62,12 +63,22 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
```

Comment:
please move the declaration back to the loop where it was before. extending the scope is error prone